pyccl/tracers.py

```diff
@@ -642,18 +675,18 @@ def add_tracer(self, cosmo, *, kernel=None,
         self._trc.append(_check_returned_tracer(ret))
 
     @classmethod
-    def from_zPower(cls, cosmo, *, A, alpha, z_min=0., z_max=6., n_chi=1024):
```

Comment:
doesn't this break the API?

Reply:
no, this is all unreleased code

Reply:
(it's in master, but it's not been released)